src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -36,12 +36,20 @@ const Body = BaseForm.extend(
         ? loc('oie.phone.verify.sms.sendText', 'login')
         : loc('oie.phone.verify.call.sendText', 'login');
       const carrierChargesText = loc('oie.phone.carrier.charges', 'login');
-      const extraCssClasses =
-        this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ?
-          'strong no-translate' : '';
+      const isPhoneNumberAvailable = this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login');
+      const extraCssClasses = isPhoneNumberAvailable ? 'strong no-translate' : '';
+      let nicknameText = isPhoneNumberAvailable ? this.model.get('nickname') : '';
+      let extraNicknameCssClasses = '';
+      if (nicknameText !== '') {
+        nicknameText = ' (' + nicknameText + ')';
+        extraNicknameCssClasses = 'strong no-translate authenticator-enrollment-nickname';
+      }
       // Courage doesn't support HTML, hence creating a subtitle here.
       this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">${sendText}
-        <span ${ extraCssClasses ? 'class="' + extraCssClasses + '"' : ''}>${this.model.escape('phoneNumber')}</span>
+        <span ${ extraCssClasses ? 'class="' + extraCssClasses + '"' : ''}>
+        ${this.model.escape('phoneNumber')}</span>
+        <span ${ extraNicknameCssClasses ? 'class="' + extraNicknameCssClasses + '"' : ''}>
+        ${nicknameText}.</span>
```

Comment:
Same as the comment left in ChallengeAuthenticatorPhoneView. 

You should probably only add the nickName span element if the nickname value exists, (it also depends on if the phone number is provided). I dont think it will cause a problem visually to have an empty span here but that may be an a11y issue.

Can you create a variable above this.add(... logic, and set the span with the nickName only if nickName is set. If it is not set, you can leave it as an empty string.

Then, in this area, you can just append the variable, so regardless on if it exists or not, it will render properly.

Reply:
Addressed.